src/components/BlogPage/BlogPage.scss

```
@@ -0,0 +1,304 @@
+      transition: all 0.4s ease-out;
+    }
+
+    &-breakable {
+      word-break: break-all;
+      display: inline-block;
+      position: relative;
+      top: -14px;
+    }
+  }
+
+  &__list {
+    margin: 0;
+    padding-left: 25px;
+
+    li {
+      list-style: none;
+
+      &::before {
+        content: '';
+        display: inline-block;
+        width: 4px;
+        height: 4px;
+        margin: 0 10px 4px -17px;
+        background-color: var(--text-service);
+        border-radius: 50%;
+      }
+    }
+  }
+
+  h3,
+  h5,
+  h6 {
+    @include font-poppins($fw-bold);
+  }
+
+  &__wrapper {
+    @include font-noto-sans();
+    @include font-scale();
+
+    padding-top: 64px;
+    line-height: 24px;
+
+    @include breakpoint($tablet-sm) {
+      padding-top: 120px;
+    }
+
+    @include breakpoint($tablet-lg) {
+      padding-top: 160px;
+    }
+  }
+
+  &__title-content {
+    @include font-poppins($fw-bold);
+    @include font-scale(x4-medium);
+
+    margin-top: 0;
+    margin-bottom: 24px;
+
+    @include breakpoint($tablet-sm) {
+      @include font-scale(large);
+    }
+  }
+
+  &__text-content {
+    @include font-noto-sans();
+    @include font-scale();
+
+    line-height: 24px;
+  }
+
+  &__chapter {
+    @include font-noto-sans($fw-bold);
+    @include font-scale();
+
+    text-transform: uppercase;
+    color: var(--text-service);
+  }
+
+  &__code {
+    display: inline-block;
+    width: 100%;
+    margin: 14px 0;
+    white-space: pre-wrap;
+
+    div[role='button'] {
+      position: absolute;
+      right: 20px;
+      top: 20px;
+    }
+
+    code {
+      @include roboto-mono();
+      @include font-scale();
+
+      position: relative;
+      display: inline-block;
+      width: 100%;
+      padding: 20px 40px 20px 20px;
+      background-color: var(--bg-light-grey);
+      border: 1px solid var(--purple-grey);
+      border-radius: 8px;
+      color: var(--text-service);
+      line-height: 1.75;
+      white-space: pre-wrap;
+    }
+
+    svg {
+      fill: var(--text-service);
+
+      &:hover {
+        fill: var(--text-light-grey);
+      }
+    }
+  }
+
+  &__code-content {
+    white-space: pre-wrap;
+  }
+
+  &__popup {
+    code {
+      @include font-scale(s-small);
+
+      padding-right: 40px;
+    }
+
+    svg {
+      @include font-scale();
+    }
+  }
+
+  &__marked-list {
+    padding-left: 30px;
+
+    li {
+      margin-bottom: 12px;
+      list-style: none;
+
+      &::before {
+        content: '';
+        display: inline-block;
+        width: 8px;
+        height: 8px;
+        margin: 0 12px 1px -17px;
+        background-color: var(--text-light-grey);
+        border-radius: 50%;
+      }
+    }
+  }
+
+  &__access {
+    h5 {
+      @include font-scale(medium);
+
+      margin: 32px 0 16px;
+    }
+
+    h6 {
+      @include font-scale(base);
+
+      margin: 16px 0 8px;
+    }
+
+    &-container {
+      display: block;
+
+      @include breakpoint($tablet-sm) {
```

For tablet please use this one. Don't use `$tablet-sm` at all. I wrote about this is Readme. Check other places
```suggestion
      @include breakpoint($tablet-sm-exact) {
```

---

@ViktorSoroka07 done